datadog_checks_base/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 * Bump the `cryptography` version to 41.0.5 ([#16083](https://github.com/DataDog/integrations-core/pull/16083))
 
+***Fixed***:
```

Comment:
Could you remove the changelog kodificaiton and use the new process?

Reply:
Done.